crates/re_viewer_context/Cargo.toml

```
@@ -40,6 +40,7 @@ lazy_static.workspace = true
 macaw.workspace = true
 ndarray.workspace = true
 nohash-hasher.workspace = true
+once_cell.workapce = true
```

Wait what, how does this pass CI?!
```suggestion
once_cell.workspace = true